cmd/ttn-lw-stack/commands/config.go

```diff
@@ -53,21 +53,23 @@ type Config struct {
 	DTC              devicetemplateconverter.Config    `name:"dtc"`
 	QRG              qrcodegenerator.Config            `name:"qrg"`
 	PBA              packetbrokeragent.Config          `name:"pba"`
+	OutputFormat     string                            `name:"output-format" yaml:"output-format" description:"Output format"`
 }
 
 // DefaultConfig contains the default config for the ttn-lw-stack binary.
 var DefaultConfig = Config{
-	ServiceBase: shared.DefaultServiceBase,
-	IS:          shared_identityserver.DefaultIdentityServerConfig,
-	GS:          shared_gatewayserver.DefaultGatewayServerConfig,
-	NS:          shared_networkserver.DefaultNetworkServerConfig,
-	AS:          shared_applicationserver.DefaultApplicationServerConfig,
-	JS:          shared_joinserver.DefaultJoinServerConfig,
-	Console:     shared_console.DefaultConsoleConfig,
-	GCS:         shared_gatewayconfigurationserver.DefaultGatewayConfigurationServerConfig,
-	DTC:         shared_devicetemplateconverter.DefaultDeviceTemplateConverterConfig,
-	QRG:         shared_qrcodegenerator.DefaultQRCodeGeneratorConfig,
-	PBA:         shared_packetbrokeragent.DefaultPacketBrokerAgentConfig,
+	ServiceBase:  shared.DefaultServiceBase,
+	IS:           shared_identityserver.DefaultIdentityServerConfig,
+	GS:           shared_gatewayserver.DefaultGatewayServerConfig,
+	NS:           shared_networkserver.DefaultNetworkServerConfig,
+	AS:           shared_applicationserver.DefaultApplicationServerConfig,
+	JS:           shared_joinserver.DefaultJoinServerConfig,
+	Console:      shared_console.DefaultConsoleConfig,
+	GCS:          shared_gatewayconfigurationserver.DefaultGatewayConfigurationServerConfig,
+	DTC:          shared_devicetemplateconverter.DefaultDeviceTemplateConverterConfig,
+	QRG:          shared_qrcodegenerator.DefaultQRCodeGeneratorConfig,
+	PBA:          shared_packetbrokeragent.DefaultPacketBrokerAgentConfig,
+	OutputFormat: "json",
```

Comment:
Put this in a new section (via empty line) to reduce diff and internal merge friendliness